macros/create_artifact_resources.sql

```diff
@@ -1,6 +1,13 @@
 {% macro create_artifact_resources() %}
 
 {% set src_dbt_artifacts = source('dbt_artifacts', 'artifacts') %}
+{% set artifact_stage = var('dbt_artifacts_stage', 'dbt_artifacts_stage') %}
+
+{% set src_results = source('dbt_artifacts', 'dbt_run_results') %}
+{% set src_result_nodes = source('dbt_artifacts', 'dbt_run_result_nodes') %}
```

Comment:
```suggestion
{% set src_run_results = source('dbt_artifacts', 'dbt_run_results') %}
{% set src_run_results_nodes = source('dbt_artifacts', 'dbt_run_results_nodes') %}
```

Just to keep the names aligned with the artifacts exactly